docs/source/releases/latest/687-change-release-note-yaml-folder-from-version-to-latest.yaml

```diff
@@ -0,0 +1,15 @@
+continuous integration:
+- description: 'test'
+  files:
+    added:
+    - ''
+    deleted:
+    - ''
+    modified:
+    - ''
+    moved:
+    - ''
+  related-issue:
+    number: 0
+    repo_url: ''
+  title: 'test'
```

Comment:
Does this need to actually be filled out?

Reply:
Yes! Filled out now